src/sentry/event_manager.py

```diff
@@ -2852,10 +2854,23 @@ def _update_user_reports_with_event_link(job: Job, group_info: GroupInfo) -> Non
         environment_id__isnull=True,
     )
 
+    for report in user_reports_without_group:
+        shim_to_feedback(
+            {
+                "name": report.name,
+                "email": report.email,
+                "comments": report.comments,
+                "event_id": report.event_id,
+                "level": "error",
+            },
+            event,
+            project,
+            FeedbackCreationSource.USER_REPORT_ENVELOPE,
```

Comment:
Maybe change this source to update_user_reports?

Reply:
why?

Reply:
(we can't tell the source origin of ths feedback, so envelope generally works and we have frontend logic for that)

Reply:
Got it. My reasoning was this "shimming" is sorta taking the place of the update_user_reports task.. but I guess we should remove that source type if/when the task isn't needed